core/API/DocumentationGenerator.php

```diff
@@ -48,9 +48,10 @@ public function __construct()
      *
      * @param bool $outputExampleUrls
      * @param string $prefixUrls
+     * @param bool $displayTitlesAsAngularDirective Set to false for the API ref doc at http://developer.piwik.org/api-reference/reporting-api where we need to display titles without using AngularJS
      * @return string
      */
-    public function getAllInterfaceString($outputExampleUrls = true, $prefixUrls = '')
+    public function getAllInterfaceString($outputExampleUrls = true, $prefixUrls = '', $displayTitlesAsAngularDirective = true)
```

Comment:
It would be better to either have a separate method for this, OR to rename it to something like "embedded or widgetized" and not that specific that will break over time and be misused over time. Ideally it was a separate method though or we would include angular wherever it is embedded.